pymars/tests/test_methods.py

```diff
@@ -77,3 +78,28 @@ def testPFA():
 	# Make sure models are the same
 	assert len(reduced_model.species()) == len(expected_model.species())
 	assert len(reduced_model.reactions()) == len(expected_model.reactions())
+
+def testDRG():
+
+	# Original model
+	path_to_original = relative_location("example_files/gri30.cti")
+	solution_object = ct.Solution(path_to_original)
+
+	# Conditions for reduction	
+	conditions =  relative_location("example_files/example_input_file.txt")
+	error = 5.0
+	target_species = ["CH4","O2"]
+	retained_species = ["CH4","O2","N2","H2O","CO2"]
+	final_error = [0]
+	epsilon_star = .5
+
+	# Run DRG
+	reduced_model = drg.run_drg(solution_object, conditions, error, target_species, retained_species, path_to_original, final_error)
+
+	# Expected answer	
+	path_to_answer = relative_location("pymars/tests/drg_gri30.cti")
+	expected_model = ct.Solution(path_to_answer)
+
+	# Make sure models are the same
+	assert len(reduced_model.species()) == len(expected_model.species())
+	assert len(reduced_model.reactions()) == len(expected_model.reactions())
```

Comment:
Rather than checking that the number of species and reactions are the same, you should check that the actual species are the same. You should be able to do this by creating a set for each of them, and checking that the sets are identical.

Reply:
You may be able to do that for reactions too.

Reply:
Changed it to compare the list of species names.